_std/defines/minimap.dm

```diff
@@ -1,6 +1,7 @@
 // Determines the minimap types the icon/area should be displayed on.
-#define MAP_DEFAULT		(1<<0)
+#define MAP_ALL			(1<<0)
```

Comment:
personal preference: `*_ALL` could be `~0`, and we save a flag spot for later :)?